question_edit/ccm.question_edit.js

```diff
@@ -113,28 +113,36 @@
         // render button to add new questions
         renderAddQuestionButton();
 
-        const saveButton = document.createElement('button');
-        const notificationSpan = document.createElement('span');
-        saveElem.appendChild(saveButton);
-        saveElem.appendChild(notificationSpan);
-
-        saveButton.setAttribute('type', 'button');
-        saveButton.className = "btn btn-info";
-        saveButton.innerText = 'Save';
-        saveButton.addEventListener('click', async () => {
-          // TODO: confirm answers are stored?
-          let entries = [];
-          Object.keys(questionData).sort().forEach( ( key ) => {
-            if ( key === 'question_ids' ) return;
-            entries.push(questionData[key]);
-          } );
+        saveData();
+
+        function saveData(){
```

Comment:
should call this renderSaveElem to make it consistent, also more accurate to what it's doing